fix: correct the AC requirement to match the correct behaviour (#2111)
vegaprotocol/specs: protocol/0019-MCAL-margin_calculator.md

**Check API:**

> - For each market and each party which has either orders or positions or has switched between margin modes on the market, the API provides the maintenance margin levels. (<a name="0019-MCAL-092" href="#0019-MCAL-092">0019-MCAL-092</a>)

Reviewer: ```suggestion
- For each market and each party which has either orders or positions in cross margin mode or has switched to cross margin mode, the API provides the maintenance margin levels. (<a name="0019-MCAL-092" href="#0019-MCAL-092">0019-MCAL-092</a>)
```
